src/main/java/org/valdroz/vscript/BaseNode.java

```diff
@@ -330,7 +330,17 @@ public Variant execute(VariantContainer variantContainer) {
                 result = Variant.fromArray(arrItems);
             }
             break;
-
+            case NT_MF_IF: {
+                List<Variant> parameterItems = Lists.newArrayList();
+                if (params != null) {
+                    params.forEach(itemNode -> parameterItems.add(itemNode.execute(variantContainer)));
+                }
+                if (parameterItems.size() != 3) {
+                    throw new EvaluationException("Number of Parameters must be 3. Sample if(true, a, b)");
+                }
+                result = Variant.fromIf(parameterItems);
```

Comment:
Variant shall not implement `if` function

Reply:
removed implementation.